In [1]:
import seaborn as sns    
titanic = sns.load_dataset("titanic")
titanic.shape

Matplotlib is building the font cache; this may take a moment.


(891, 15)

In [4]:
titanic.dtypes
titanic.isna().sum()

survived         0
pclass           0
sex              0
age            177
sibsp            0
parch            0
fare             0
embarked         2
class            0
who              0
adult_male       0
deck           688
embark_town      2
alive            0
alone            0
dtype: int64

In [9]:
titanic["sex"].value_counts()


sex
male      577
female    314
Name: count, dtype: int64

In [7]:
titanic["class"].value_counts() 

class
Third     491
First     216
Second    184
Name: count, dtype: int64

In [8]:
titanic["embark_town"].value_counts()

embark_town
Southampton    644
Cherbourg      168
Queenstown      77
Name: count, dtype: int64

## Constats

- Le jeu de données contient 891 passagers, et 177 n'ont pas d'âge connu 
- La majorité des passagers voyageait en 3e classe (491 sur 891, soit un peu plus de la moitié).
- Les colonnes `class` et `pclass` donnent la même information donc sont redondants 